plugins/externalwrench/src/ExternalWrench.cc

```diff
@@ -106,7 +109,7 @@ void ExternalWrench::setVisual()
     visualMsg.set_cast_shadows(false);
 }
 
-bool ExternalWrench::setWrench(physics::ModelPtr& _model,yarp::os::Bottle& cmd)
+bool ExternalWrench::setWrench(physics::ModelPtr& _model,yarp::os::Bottle& cmd, double& simulationUpdatePeriod, bool& wrenchSmoothing)
```

Comment:
Probably, passing input arguments with a `const` is preferable.

Reply:
Done https://github.com/Yeshasvitvs/gazebo-yarp-plugins/commit/b33cfcfca4a80c85ea300ba6cc3ec19a31ba8f8e